sgf/fetch_dataset.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
```

Comment:
Can you put a comment in here that these games are public domain?

Reply:
Done